profiles/common/modules/features/multisite_twitterblock/multisite_twitterblock.module

```diff
@@ -402,11 +402,11 @@ function multisite_twitterblock_form_node_form_alter(&$form, $form_state) {
 
 /**
  * Provides a settings subform.
- * 
+ *
  * @param array $form
  *   Rendable array.
  */
-function _multisite_twitterblock_entity_subform(&$form, $fpfis_twitter_override = array()) {
+function _multisite_twitterblock_entity_subform(array &$form = array(), $fpfis_twitter_override = array()) {
```

Comment:
The `$form` array is not optional, so don't append ` = array()` to it. Prefixing it with `array ` is a very good practice though.